docker/release/Dockerfile

```diff
@@ -33,6 +33,7 @@ ENV OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_LOCAL_REPOSITORY="/usr/
 ENV OSTK_PHYSICS_ENVIRONMENT_EPHEMERIDES_SPICE_MANAGER_LOCAL_REPOSITORY="/usr/local/share/OpenSpaceToolkit/Physics/environment/ephemerides/spice"
 ENV OSTK_PHYSICS_ENVIRONMENT_GRAVITATIONAL_EARTH_MANAGER_LOCAL_REPOSITORY="/usr/local/share/OpenSpaceToolkit/Physics/environment/gravitational/earth"
 ENV OSTK_PHYSICS_ENVIRONMENT_MAGNETIC_EARTH_MANAGER_LOCAL_REPOSITORY="/usr/local/share/OpenSpaceToolkit/Physics/environment/magnetic/earth"
+ENV OSTK_PHYSICS_ENVIRONMENT_ATMOSPHERIC_EARTH_MANAGER_LOCAL_REPOSITORY="/usr/local/share/OpenSpaceToolkit/Physics/environment/atmospheric/earth"
 
```

Comment:
Why don't we need to the `OSTK_PHYSICS_DATA_MANIFEST_LOCAL_REPOSITORY` here?

Reply:
We technically don't _need_ any of these here. If these aren't set, then the default value is used. (it's something like `./open-space-toolkit/data/manifest.json`)

It's usually totally fine for it to use the default, but it's definitely better to have it here for consistency! Good catch.

Reply:
so then it should be added?

Reply:
Yep! Might as well